src/runtime/internal/dev.ts

```diff
@@ -155,6 +156,10 @@ export class SvelteComponentDev extends SvelteComponent {
 			throw new Error("'target' is a required option");
 		}
 
+		if (!options.target && !current_component) {
+			throw new Error("Multiple Svelte instances detected. Please ensure only one instance exists, or set the 'target' option");
```

Comment:
Like the better error message! Should we be wordier here and explain in more detail why this can happen? Something like "This can happen if you use Svelte components which are precompiled to JavaScript. Ensure to use the raw Svelte version of these components".

Reply:
Yeah that sounds like a better explanation, I'll rephrase it to something like that.